fastlane/actions/gh_pages.rb

```diff
@@ -14,26 +14,33 @@ def self.run(params)
         UI.user_error!("Missing `_versions` file") unless File.exists?("_versions")
         UI.user_error!("Missing `.jazzy.yaml` file") unless File.exists?(".jazzy.yaml")
         module_version = YAML.load(File.read(".jazzy.yaml"))["module_version"]
+        array = module_version.split(/[.]/)
+        short_version = array[0] + "." + array[1]
+        module_version = short_version
```

Comment:
This action is also used by [IAM](https://github.com/rakutentech/ios-inappmessaging/blob/master/bitrise.yml#L60) so I think this (and the module_name prefix) change will mess up future deploys compared to the current IAM docs versioning URL naming https://rakutentech.github.io/ios-inappmessaging/6.0.0/index.html ?

Instead we can just use the full n.n.n version and remove the module name prefix. Keeping only the latest patch version docs is a legacy from the DRC docs portal. Sorry if this change was due to my investigation branch.